.github/workflows/container-orchestration.yaml

```diff
@@ -132,6 +132,23 @@ jobs:
         working-directory: ./containers/${{env.CONTAINER}}/tests/integration
         run: |
           python -m pytest -m "integration"
+
+      - name: Get docker logs
```

Comment:
was this for debugging? or something we want all the time? 30 days retention feels kinda long for this